bin/generate.py

```diff
@@ -709,18 +707,22 @@ def validate_ans_and_out(
             bar.error("No .out file was generated!")
             return False
 
-        answer_validator_hashes = {**testcase.validator_hashes(validate.AnswerValidator, bar)}
-        if all(h in meta_yaml["answer_validator_hashes"] for h in answer_validator_hashes):
-            return True
+        ans_out_validator_hashes = testcase.validator_hashes(validate.AnswerValidator, bar).copy()
+        output_validator_hashes = testcase.validator_hashes(validate.OutputValidator, bar)
 
         mode = validate.Mode.ANSWER
-        if testcase.root in config.INVALID_CASE_DIRECTORIES:
+        if testcase.root == ["invalid_answer"]:
             mode = validate.Mode.INVALID
-        elif testcase.root == "valid_output":
+        elif testcase.root in ["valid_output", "invalid_output"]:
+            ans_out_validator_hashes.update(output_validator_hashes)
             mode = validate.Mode.VALID_OUTPUT
```

Comment:
I think this change broke the CI? The error I'm seeing, is:
```
Generate: invalid_output/wrong_answer               default_output_validator.cpp: rejected
default_output_validator.cpp: Got: 1, wanted: 0
-> /tmp/bapctools_5d438c/identity/links/8885fb/testcase.out
```
The entry for `invalid_output/wrong_answer` in `generators.yaml` is:
```
  invalid_output:
    data:
      "wrong_answer":
        in: "0"
        ans: "0"
        out: "1"
```
So, it should be correct that the answer is wrong. However, the test case (with root `invalid_output`) now receives `Mode.VALID_OUTPUT`, but that should be `Mode.INVALID`, right?